app/assets/stylesheets/application.tailwind.css

```
@@ -22,28 +22,42 @@
+    }
+
+    50% {
+      transform: translateY(0);
+    }
   }
 }
 
-  @utility animate-bounce-slow {
-    animation: bounce-slow 2s ease-in-out infinite;
-  }
+@utility animate-bounce-slow {
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)